openpype/hosts/blender/plugins/publish/extract_blend.py

```diff
@@ -202,7 +202,13 @@ def _process_resources(
             # Store the hashes from hash to destination to include in the
             # database
             # NOTE Keep source hash system in case HARDLINK system works again
-            texture_hash = source_hash(sourcepath)
+            try:
+                texture_hash = source_hash(sourcepath)
+            except FileNotFoundError:
+                self.log.warning(
+                    f"Path not found on disk, skipping remapping: {sourcepath}"
+                )
```

Comment:
Isn't it better to keep the original error message?
```suggestion
            except FileNotFoundError as e:
                self.log.warning(e)
```